src/sources/soundsource.cpp

```diff
@@ -45,45 +32,40 @@ QString SoundSource::getTypeFromUrl(const QUrl& url) {
 
 //static
 QString SoundSource::getTypeFromFile(const QFileInfo& fileInfo) {
-    const QString fileSuffix = fileInfo.suffix();
-    const QString fileType = fileTypeFromSuffix(fileSuffix);
-    DEBUG_ASSERT(!fileType.isEmpty() || fileType == QString{});
-    const QMimeType mimeType = QMimeDatabase().mimeTypeForFile(
+    const QString fileSuffix = fileInfo.suffix().toLower().trimmed();
+
+    if (fileSuffix == "opus") {
+        // opus has mime type "audio/ogg" which will be decoded with the SoundSourceOggVobis()
```

Comment:
I think this is because "ogg" is a container and "vorbis" is the compression algorithm.

Reply:
Yes, the OGG container can include Speex, Vorbis, Opus, FLAC, OggPCM. 
The Ogg magic numbers are: 
`OggS...........v.....v......vorbis` is Vorbis
`OggS.............l..........Opus` is Opus

`fLaC..."`is the magic number for native FLAC. I think we do not support Ogg FLAC which will probably also start with `OggS`